src/script_opt/ZAM/ZInst.h

```diff
@@ -317,6 +329,7 @@ class ZInstAux
 			slots = ints = new int[n];
 			constants = new ValPtr[n];
 			types = new TypePtr[n];
+			is_managed = new bool[n];
```

Comment:
Yee-up, I debated doing just that. I held off because it would have made the PR bigger still, but I imagine the next time I touch `ZInstAux` I'll bite the bullet and do it.

Reply:
@timwoj Can this be resolved?